test/test-debuglet.js

```diff
@@ -120,6 +122,61 @@ describe('Debuglet', function() {
       debuglet.start();
     });
 
+    it('should use GCLOUD_PROJECT in lieu of config.projectId', function(done) {
+      process.env.GCLOUD_PROJECT = '11020304f2934-b';
```

Comment:
The test happens to already `delete process.env.GCLOUD_PROJECT` before each test case. But I can add saving/restoring logic there.